packages/ember-data/lib/system/store.js

```diff
@@ -440,9 +440,32 @@ Store = Ember.Object.extend({
     store.find('person', { page: 1 });
     ```
 
-    This will ask the adapter's `findQuery` method to find the records for
-    the query, and return a promise that will be resolved once the server
-    responds.
+    This will call the private method `findQuery`, which delegates the query to the adapter. The adapter then makes a call to the server, which will return a `RecordArray` when the promise resolves.
```

Comment:
> This will call the private method `findQuery`

I'm not sure if we want to document the internal behavior like this. Is there a specific reason why you thought it would be useful to call out the store's `findQuery` method?